docs/about/community.md

```diff
@@ -1,22 +1,40 @@
 ---
 id: community
-title: Community Engagement
 custom_edit_url: null
 ---
+
 # Community Partners
 
 ## About
+
 We welcome the participation of anyone interested in contributing to the projects in our portfolio. You can work on open issues on projects in our portfolio, create a new issue, or e-mail us if you want to be more involved.
 
+Join the conversation in the Open Source with SLU [Slack workspace](https://join.slack.com/t/oswslu/shared_invite/zt-24f0qhjbo-NkSfQ4LOg5wXxBdxP4vzfA).
+
+## Overview
+
+* Join our Slack workspace.
+* Contribute changes to an Open Source with SLU project.
+* Add your name to the Contributors list.
+* After making five contributions, you can be added to the `oss-slu` team in GitHub by contacting one of the program staff.
+
+## Getting Startted
+
+Anyone interested in getting involved with Open Source with SLU projects can sign up by opening a [Pull Request (PR)](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) on the [contributors](contributors.md) page in this website. The PR will be merged when it includes a link to a merged ticket from a project in the [portfolio](../portfolio.md) that they made contribution to, and the PR has been approved by the current tech lead on that project.
+
```

Comment:
Maybe add a detail on what exactly they need to add to the contributors.md (students doing this for the first time may be confused, especially since there are no names on that list yet).

Or, maybe we can have a PR template for "I want to be a contributor", which specifies to:
1. Add your name to the list in contributors.md
2. Add a link to merged ticket (by 'ticket', did you mean a PR or a closed issue?)

In the overview, we say that contributors need to make 5 contributions to be added to the list. Are those contributions issues or PR? Do new contributors need to add 5 links to tickets in their PRs here?

Reply:
I like all this. "Yes, and"